.circleci/config.yml

```diff
@@ -21,9 +21,33 @@ jobs:
             pip install -t lib/ -r requirements.txt
             pip install nose nosegae nose-exclude
             nosetests --with-gae --gae-lib-root=`gcloud info --format="value(installation.sdk_root)"`/platform/google_appengine --exclude-dir=lib
-          
+
+  tag:
+    docker:
+      - image: alpine/git
+
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "ab:2d:12:de:d1:ac:36:0b:6a:02:92:3f:2c:49:98:c9"
+      - checkout
+      - run:
+          when: on_success
+          command: |
+            echo 'export TAG_NAME=$(date +%s)' >> $BASH_ENV
```

Comment:
does BASH_ENV come from the circle env vars, or are you expecting to pass it in somehow?

Reply:
i think it's being created within the job. was following these instructions: https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-shell-command